test/OpenTelemetry.Tests/Internal/CallerArgumentExpressionAttributeTests.cs

```diff
@@ -0,0 +1,50 @@
+// <copyright file="CallerArgumentExpressionAttributeTests.cs" company="OpenTelemetry Authors">
```

Comment:
I guess the general practice in this repo is to have the test file name reflecting the source file name? (e.g. in this case it'll be `GuardTests.cs`)

Reply:
General practice is also one class per file. ;)
I have no problem moving this to the GuardTests.cs. I'll get that done today :)